.github/workflows/deno.yml

```
@@ -0,0 +1,53 @@
+name: Deno
+
+on:
+  push:
+    tags:
+      - "13.*"
```

Will this line need updated when we bump major versions here?

---

Yep. The `push` event is fired after creating new tags and new branches. So I want to make sure that it's fired only on create a new version tag. To include all tags, we can try with `"*"`.

---

Yeah, I never tag things that aren't releases, so `*` will work.
```suggestion
      - "*"
```